packages/protocol/contracts/L1/libs/LibProving.sol

```diff
@@ -202,13 +198,8 @@ library LibProving {
             | (uint256(evidence.parentGasUsed) << 64)
             | (uint256(evidence.gasUsed) << 32);
 
-        // Also hash configs that will be used by circuits
-        inputs[9] = uint256(config.blockMaxGasLimit) << 192
```

Comment:
@Brechtpd  I feel like we do not need to hash configs in the instance. Let me know what you think.